docs/standalone-transaction.md

```diff
@@ -5,7 +5,7 @@ First, import the python and application specific libraries and set the static v
 ``` python
 import logging
 
-from aea.crypto.fetchai import FetchAICrypto
+from aea.configurations.constants import FETCHAI
```

Comment:
Ugh we spent effort removing this, now we are introducing it. Just import the plugin please. aea configurations constants should eventually not contain these.

Reply:
done